python/cugraph/cugraph/tests/link_prediction/test_woverlap.py

```diff
@@ -14,158 +14,39 @@
 import gc
 
 import pytest
-import scipy
-import numpy as np
-
-import cudf
+import networkx as nx
 import cugraph
-from cudf.testing import assert_series_equal
 from cugraph.testing import utils, UNDIRECTED_DATASETS
 
 
+print("Networkx version : {} ".format(nx.__version__))
```

Comment:
I'm thinking we should start removing these messages. We now have a proper test dependency that enforce specific NX versions that make these messages not useful for almost all cases.